lib/resources.js

```diff
@@ -336,10 +339,11 @@ async function _updateAcquisitionRecord(
     'acquisition.tokenized': tokenized
   };
   const now = Date.now();
+  const expirationDate = new Date(expires);
```

Comment:
@mandyvenables you mentioned that if `expires` is undefined, that bad things happen:

If the value of expires is undefined, the Date Object is set to 1970/01/01 and is inserted into the database then removed by the mongo expiration process.

Did you mention this because there is a possibility that `expires` is undefined in this code path?

Reply:
Btw, my new upcoming code that I will rebase on top of this will initialize `expires` to `new Date()` ... so we could do that here if it helps.

Reply:
Or we can just wait for my follow-on PR to handle that.

Reply:
@mattcollier I am not sure if it's actually a bad thing. In the use case we are using this library for, I don't believe an undefined value is possible. [Here is where that value is computed.](https://github.com/digitalbazaar/bedrock-resource-restriction/blob/main/lib/ResourceTokenizer.js#L404)

The question is: if `expires` is undefined, does that mean it should never expire, or expire immediately? @dlongley ?

Reply:
Undefined means it should never expire. But -- I'd say don't worry about that because my follow-on PR will deal with it.